aws_doc_sdk_examples_tools/snippets.py

```
@@ -138,12 +145,17 @@ def parse_snippets(
 def find_snippets(file: Path, prefix: str) -> Tuple[Dict[str, Snippet], MetadataErrors]:
     errors = MetadataErrors()
     snippets: Dict[str, Snippet] = {}
-    with open(file, encoding="utf-8") as snippet_file:
-        try:
-            snippets, errs = parse_snippets(snippet_file.readlines(), file, prefix)
-            errors.extend(errs)
-        except UnicodeDecodeError as err:
-            errors.append(MetadataUnicodeError(file=str(file), err=err))
+    try:
+        with open(file, encoding="utf-8") as snippet_file:
+            try:
+                snippets, errs = parse_snippets(snippet_file.readlines(), file, prefix)
+                errors.extend(errs)
+            except UnicodeDecodeError as err:
+                errors.append(MetadataUnicodeError(file=str(file), err=err))
+    except FileNotFoundError:
+        pass
```

I'm uneasy about silently passing on an exception unless there's a good reason. Maybe log at DEBUG level at minimum?